nekoyume/Assets/_Scripts/Game/Game.cs

```diff
@@ -659,5 +665,85 @@ private string GetMessage(string logString, string stackTrace)
 
             return msg;
         }
+#if UNITY_EDITOR
+        private void EditorTests()
```

Comment:
How about going to `Cheat.cs` for the test function?

Reply:
I'll remove this method because it just need to develop test.